mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/Connection.cs

```diff
@@ -810,6 +815,10 @@ class PacketReader {
 				get; set;
 			}
 
+			public string ErrorMsg {
+				get; set;
```

Comment:
```suggestion
				get; internal set;
```

Reply:
I'm not 100% confident about my suggestion here (and the other `internal set`).  If it makes sense to keep this publicly writeable in the context of debugger-libs, please ignore.